src/binary-reader-objdump.cc

```diff
@@ -908,30 +916,28 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeBlockSig(Type sig_type) {
   return Result::Ok;
 }
 
-enum class InitExprType {
-  I32,
-  F32,
-  I64,
-  F64,
-  V128,
-  Global,
-  FuncRef,
-  // TODO: There isn't a nullref anymore, this just represents ref.null of some
-  // type T.
-  NullRef,
```

Comment:
I wonder if that fact that FuncRef and NullRef are undefined actually just about us missing test cases for them.   I guess it doesn't hurt to land this either way.